### PR TITLE
added kwargs to peek() to maintain consistency

### LINKED ISSUE #3486: consistency for `.peek()`to take/not take kwargs for plotting
At the moment some TimeSeries sources allow for matplotlib `kwargs` in the `.peek()` while others do not. Map does allow it i.e. kwargs are allowed (i.e. any additional `imshow` keywords can be used in `map.peek()`.

Following from #3378 it seems that some TimeSeries sources allow it (i.e. noaa, lyra and eve) while others do now (i.e. the rest). 

I guess the question is which way do we want to have it - but we just have to be consistent!

Thoughts @dstansby ?

### PR DESCRIPTION
**Pull request by aribalam**
### Description

Added kwargs to `peek()` function of some of the TimeSeries sources to maintain consistency with other TimeSeries sources and all other Maps.

Fixes #3486

### CODE REVIEW

**Comment by aribalam** [comment]
@dstansby Thanks for pointing this out. I have made the required changes.

File: sunpy/timeseries/sources/fermi_gbm.py [hunk 1]
```diff
@@ -71,6 +71,11 @@ def peek(self):
             import sunpy.data.sample
             gbm = sunpy.timeseries.TimeSeries(sunpy.data.sample.GBM_TIMESERIES, source='GBMSummary')
             gbm.peek()
+
+        Parameters
+        ----------
+        **kwargs : `dict`
+        Additional plot keyword arguments that are handed to `axes.plot` functions
```
Comment (nabobalis) [inline sunpy/timeseries/sources/fermi_gbm.py]:
```suggestion
            Additional plot keyword arguments that are handed to `axes.plot` functions
```

File: sunpy/timeseries/sources/norh.py [hunk 2]
```diff
@@ -66,6 +66,11 @@ def peek(self):
             import sunpy.timeseries
             norh = sunpy.timeseries.TimeSeries(sunpy.data.sample.NORH_TIMESERIES, source='NoRH')
             norh.peek()
+
+        Parameters
+        ----------
+        **kwargs : `dict`
+        Additional plot keyword arguments that are handed to `axes.plot` functions
```
Comment (nabobalis) [inline sunpy/timeseries/sources/norh.py]:
```suggestion
            Additional plot keyword arguments that are handed to `axes.plot` functions
```

File: sunpy/timeseries/sources/rhessi.py [hunk 3]
```diff
@@ -77,6 +77,8 @@ def peek(self, title="RHESSI Observing Summary Count Rate"):
         ----------
         title : `str`
             The title of the plot.
+        **kwargs : `dict`
+        Additional plot keyword arguments that are handed to `axes.plot` functions
```
Comment (nabobalis) [inline sunpy/timeseries/sources/rhessi.py]:
```suggestion
            Additional plot keyword arguments that are handed to `axes.plot` functions
```

**Pull request merged**